apps/ligo-virgo/src/assets/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "display": "standalone",
+    "icons": [
+        {
+            "sizes": "192x192",
```

Comment:
I think it is better to have 256x256 and 144x144, as recommended in the pwa documentation before. Don't know if this changed these years.

Reply:
No, just use larger one of this icon is ok